apps/google-analytics-4/frontend/src/components/config-screen/assign-content-type/AssignContentTypePage.tsx

```diff
@@ -0,0 +1,170 @@
+import { useCallback, useEffect, useState } from 'react';
+import { Paragraph, Stack, Subheading } from '@contentful/f36-components';
+import {
+  AppInstallationParameters,
+  ContentTypeEntries,
+  ContentTypeEntry,
+  AllContentTypes,
+} from 'types';
+import AssignContentTypeCard from 'components/config-screen/assign-content-type/AssignContentTypeCard';
+import omitBy from 'lodash/omitBy';
+import sortBy from 'lodash/sortBy';
+import { useSDK } from '@contentful/react-apps-toolkit';
+import { AppExtensionSDK } from '@contentful/app-sdk';
+import { ContentTypeProps, createClient } from 'contentful-management';
+
+const AssignContentTypePage = () => {
+  const [parameters, setParameters] = useState<AppInstallationParameters>(
+    {} as AppInstallationParameters
+  );
+  const [allContentTypes, setAllContentTypes] = useState<AllContentTypes>({} as AllContentTypes);
+  const [contentTypeEntries, setContentTypeEntries] = useState<ContentTypeEntries>(
+    {} as ContentTypeEntries
+  );
+
+  const sdk = useSDK<AppExtensionSDK>();
+
+  const onConfigure = useCallback(async () => {
+    const currentState = await sdk.app.getCurrentState();
+
+    const newParameters = Object.assign(
+      {},
+      parameters,
+      omitBy(
```

Comment:
I know this is far from the only place we use lodash, so I don't mean to force this discussion here, 

BUT

work like this could be handled by some of the [ES6 arrow functions ](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter) on the keys of the object.

Probably just interested in checking if people are interested in making the switch, doesn't need to happen in this PR

Reply:
It's a good question -- this will be refactored in some of the pending work for this config page and we will revisit then!